crates/hc_sandbox/tests/cli.rs

```diff
@@ -657,8 +834,7 @@ async fn shutdown_sandbox(mut child: Child) {
         nix::sys::signal::kill(Pid::from_raw(pid as i32), Signal::SIGINT)
             .expect("Failed to send SIGINT");
 
-        let exit_code = child.wait().await.unwrap();
-        assert!(exit_code.success());
+        child.wait().await.unwrap();
```

Comment:
Is asserting the exit code no longer possible?

Reply:
Oh this is interesting. It doesn't actually give you back the exit status here, it gives you back the wait status, which is different. So if the process happens to shut down before you `wait` on it, then you get a non-zero status back to tell you the process was already gone. This was one of the reasons for flaky tests that I was battling for a few days with this PR. That and processes not shutting down and too many tests launching in parallel.

I should have commented on this, but yeah it's not reliably useful to assert this in the test as far as I understand